doc/source/serve/production-guide/config.md

```diff
@@ -0,0 +1,200 @@
+
+Using the `FruitStand` deployment graph example:
+
+```console
+$ ls
+fruit.py
+
+$ serve build fruit:deployment_graph -o fruit_config.yaml
+
+$ ls
+fruit.py
+fruit_config.yaml
+```
+
+(fruit-config-yaml)=
+
+The `fruit_config.yaml` file contains:
+
+```yaml
+import_path: fruit:deployment_graph
+
+runtime_env: {}
+
+deployments:
+
+- name: MangoStand
+  num_replicas: 2
+  route_prefix: null
+  max_concurrent_queries: 100
+  user_config:
+    price: 3
+  autoscaling_config: null
+  graceful_shutdown_wait_loop_s: 2.0
+  graceful_shutdown_timeout_s: 20.0
+  health_check_period_s: 10.0
+  health_check_timeout_s: 30.0
+  ray_actor_options: null
+
+- name: OrangeStand
+  num_replicas: 1
+  route_prefix: null
+  max_concurrent_queries: 100
+  user_config:
+    price: 2
+  autoscaling_config: null
+  graceful_shutdown_wait_loop_s: 2.0
+  graceful_shutdown_timeout_s: 20.0
+  health_check_period_s: 10.0
+  health_check_timeout_s: 30.0
+  ray_actor_options: null
+
+- name: PearStand
+  num_replicas: 1
+  route_prefix: null
+  max_concurrent_queries: 100
+  user_config:
+    price: 4
+  autoscaling_config: null
+  graceful_shutdown_wait_loop_s: 2.0
+  graceful_shutdown_timeout_s: 20.0
+  health_check_period_s: 10.0
+  health_check_timeout_s: 30.0
+  ray_actor_options: null
+
+- name: FruitMarket
+  num_replicas: 2
+  route_prefix: null
+  max_concurrent_queries: 100
+  user_config: null
+  autoscaling_config: null
+  graceful_shutdown_wait_loop_s: 2.0
+  graceful_shutdown_timeout_s: 20.0
+  health_check_period_s: 10.0
+  health_check_timeout_s: 30.0
+  ray_actor_options: null
+
+- name: DAGDriver
+  num_replicas: 1
+  route_prefix: /
+  max_concurrent_queries: 100
+  user_config: null
+  autoscaling_config: null
+  graceful_shutdown_wait_loop_s: 2.0
+  graceful_shutdown_timeout_s: 20.0
+  health_check_period_s: 10.0
+  health_check_timeout_s: 30.0
+  ray_actor_options: null
+
+```
+
+Note that the `runtime_env` field will always be empty when using `serve build` and must be set manually.
+
+## Overriding deployment settings
+
+Settings from `@serve.deployment` can be overriden with this Serve config file. The order of priority is (from highest to lowest):
+
+1. Config File
+2. Deployment graph code (either through the `@serve.deployment` decorator or a `.set_options()` call)
+3. Serve defaults
+
+For example, if a deployment's `num_replicas` is specified in the config file and their graph code, Serve will use the config file's value. If it's only specified in the code, Serve will use the code value. If the user doesn't specify it anywhere, Serve will use a default (which is `num_replicas=1`).
+
+Keep in mind that this override order is at the settings-level. For example, if a user has a deployment `ExampleDeployment` with the following decorator:
```

Comment:
```suggestion
Keep in mind that this override order is applied separately to each individual setting. For example, if a user has a deployment `ExampleDeployment` with the following decorator:
```

I was a bit confused by the original wording since "settings" can be a single setting or multiple. English is weird :D